cloudinstall/charms/__init__.py

```diff
@@ -104,6 +104,9 @@ class CharmBase:
     menuable = False
     subordinate = False
     openstack_release_min = 'i'
+    # Set this if a plugin charm overrides a 'system' installed
+    # charm
+    tainted = False
```

Comment:
So i was thinking for support requests being able to check if the system installed charm was used or if someone overwrote it with their own